compiler/src/dotty/tools/dotc/plugins/Plugin.scala

```diff
@@ -50,7 +51,19 @@ trait StandardPlugin extends Plugin {
    *  @param options commandline options to the plugin.
    *  @return a list of phases to be added to the phase plan
    */
-  def init(options: List[String]): List[PluginPhase]
+  @deprecated("`init` does not allow to access `Context`, use `initialize` instead.", since = "3.5.0")
```

Comment:
What you want is `@deprecatedOverride` ;)
(and also probably `@deprecated`, but for a different reason)